src/ontology/foodon-edit.owl

```diff
@@ -18405,6 +18405,59 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003924 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003924">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001009"/>
```

Comment:
corndog can't be a child of hotdog.  It can go under "multicomponent meat product" for now though.

Reply:
Right, but as per many sources, corndog is usually categorized as a hotdog. https://en.wikipedia.org/wiki/Corn_dog. Considering that, Shouldn't 'corndog' be a child of both classes, 'hot dog' and 'multicomponent meat product'

Reply:
We have to distinguish a "hot dog" that one buys in a package from a store refrigerator, from a "hot dog in a bun" which vendors serve.  But a corndog is neither of these and can't be a child of them.  A corndog has a hotdog inside it.

Reply:
or as wikipedia says, usually has a hotdog inside it, but sometimes another kind of sausage I guess.

Reply:
Right, I think that makes sense. For now, I'll add that under 'multicomponent meat product'. Thanks

Reply:
Who let the dogs out? (sorry.)